### PR TITLE
Setup connection between front end back end

### PR DESCRIPTION
**Pull request by reyven-ai**

### CODE REVIEW

File: backend/src/codegen.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+generates:
+  src/graphql/generated.tsx:
+    plugins:
+      - 'typescript'
+      - 'typescript-operations'
+      - 'typescript-react-apollo'
+    config:
+      withHooks: true
+      withHOC: false
+      withComponent: false
```
Comment (artjom-mashanin) [inline backend/src/codegen.yml]: You don't need this here — it's codegen file for react, you don't need it in backend

File: frontend/src/components/apollo-client.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { ApolloClient, InMemoryCache } from "@apollo/client";
```
Comment (artjom-mashanin) [inline frontend/src/components/apollo-client.ts:1]: apollo-client file shouldn't be in components folder — find a different spot for it 😉

**Review by artjom-mashanin — COMMENTED** [review]
Good job, Ven!

Left 2 minor comments

File: backend/src/codegen.yml (hunk 1 above)
Comment (reyven-ai) [reply backend/src/codegen.yml]: Done!

File: frontend/src/components/apollo-client.ts (hunk 2 above)
Comment (reyven-ai) [reply frontend/src/components/apollo-client.ts:1]: Done!

**Review by artjom-mashanin — APPROVED** [review]
Great job 🥇

**Pull request merged**